[TASK] Update RTE page (#195)
TYPO3-Documentation/TYPO3CMS-Tutorial-Editors: Documentation/ContentElements/RichTextEditor/Index.rst

..  include:: /Includes.rst.txt
..  _rte:

====================
The rich text editor
====================

..  image:: /Images/ManualScreenshots/ContentElements/RegularText.png
     :alt: Content element Regular Text in the TYPO3 backend
     :class: img-thumbnail float-end ms-1
     :width: 250

Start by `Creating a content element <https://docs.typo3.org/permalink/t3editors:content-creating>`_

> of type "Text & Images", "Text & Media", or "General Text". You can find them in the

Reviewer: Text on your image does not match value in text:

```suggestion
of type "Text & Images", "Text & Media", or "Regular Text Element". You can find them in the
```

tab `Typical page content <https://docs.typo3.org/permalink/t3editors:content-typical>`_.

The TYPO3 text editing interface is not much different from a word processor.
The options available in the toolbars vary depending on how your TYPO3
installation is configured.

..  figure:: /Images/ManualScreenshots/ContentElements/Text/RichTextEditor.png
     :alt: Screenshot of a typical rich text editor in the TYPO3 backend

     The options you see depend on the configuration of your site

You can format text, set the alignment, insert tables and symbols and add
internal and external links to text.

..  _rte-linking:

Creating a link
===============

Select the text you want to link.

Click the :guilabel:`Link` icon in the toolbar or use the shortcut

> :kbd:`Ctrl + K`.

Reviewer: That's the same shortcut as for global search. This shortcut will only work, if cursor stays in RTE. If it is somewhere else the global search will open.
```suggestion
:kbd:`Ctrl + K` or :kbd:`Cmd + K`.
```

Author: Yep, should we communicate this and if so how?

Reviewer: I have posted a question in #typo3-cms

Author: I have added a note

> :kbd:`Ctrl + K` or :kbd:`Cmd + K`.

Author: ```suggestion
:kbd:`Ctrl + K` or :kbd:`Cmd + K`. This shortcut only works when your focus is within the editor. Outside the editor the same shortcut would open the life search.
```

..  figure:: /Images/ManualScreenshots/ContentElements/Text/Link.png
    :alt: Linking the highlighted text in the TYPO3 rich text editor

The :guilabel:`Link Browser` window displays.

This window lets you link to an internal page, file, folder, external
URL, email address or phone number.

..  _rte-linking-internal:

Internal links
--------------

..  figure:: /Images/ManualScreenshots/Link/LinkBrowser.png
    :alt: The TYPO3 CMS link browser

You can link to an internal file, folder, page or content element on a page.

On the :guilabel:`Page` tab, select the page in the page tree that you want
to link to, then click the :guilabel:`Link to ...` button.

You can also link directly to a content element. In that case an anchor
(link with a hashtag) is used so your readers will automatically jump to the
desired content element.

You can also link to uploaded files, for example PDF files:

..  figure:: /Images/ManualScreenshots/Link/LinkBrowserFile.png
    :alt: Linking to a file for download in TYPO3

..  _rte-linking-external:

External links
--------------

> #.  In the :guilabel:`Link Browser` window, go to the :guilabel:`*External URL` tab.

Reviewer: ```suggestion
#.  In the :guilabel:`Link Browser` window, go to the :guilabel:`External URL` tab.
```
